packages/jsii-pacmak/test/expected.jsii-calc-base/sphinx/_scope_jsii-calc-base.rst

```diff
@@ -196,3 +196,10 @@ BaseProps (interface)
       :type: string *(abstract)*
 
 
+   .. py:attribute:: foo
+
+      *Inherited from* :py:attr:`@scope/jsii-calc-base-of-base.VeryBaseProps.foo`
```

Comment:
Is it not more correct to say "inherited from <class>" instead of "inherited from <method>" ?

Reply:
But... but they're the same string...

![](https://media.giphy.com/media/CPskAi4C6WLHa/giphy.gif)

Reply:
Oh well. I wrote something between `<` and `>` but got eaten by the MarkDown rendered. Shoulda guessed.

I meant to say, "inherited from $CLASS" seems more correct than "inherited from $METHOD".

Fuuuuu--- that gif is distracting!

Reply:
That's what I kinda guessed when I saw your first comment. I wanted the link to be to the parent class' method/property, and not to the class itself... But what I can do is set a custom label on the link, so it'll be the best of both.